Univers/SuperamasDeLaVierge/GroupeLocal/VoieLactée/BrasOrion/SystèmeSolaire/ZoneHabitable/Terre/Amérique/US/Texas/Dallas/MaMaison/MonOrdinateur/home/sebastien/document/movaicode/laClassADallas.java

```diff
@@ -0,0 +1,16 @@
+package Univers.SuperamasDeLaVierge.GroupeLocal.VoieLactée.BrasOrion.SystèmeSolaire.ZoneHabitable.Terre.Amérique.US.Texas.Dallas.MaMaison.MonOrdinateur.home.sebastien.document.movaicode;

```

Comment:
Je pense qu'il peut y avoir un soucis suivant les systèmes sur lesquels tu fais tourner ton code à cause des accents dans tes noms de package :/ De plus ça mériterait d'être internationalisé, superamas devient supercluster:

```suggestion
package Universe.VirgoSuperCluster.LocalGroup.MilkyWay.OrionArm.SolarSystem.InhabitableArea.Earth.America.US.Texas.Dallas.MyHome.MyComputer.home.sebastian.document.movaicode;
```

Reply:
Yep pas mal je change ça